bin/node/src/aleph_node_rpc.rs

```diff
@@ -248,6 +272,69 @@ where
     fn ready(&self) -> RpcResult<bool> {
         Ok(!self.sync_oracle.is_offline() && !self.sync_oracle.is_major_syncing())
     }
+
+    async fn validator_network_info(
+        &self,
+    ) -> RpcResult<HashMap<AccountId, ValidatorAddressingInfo>> {
+        let mut info = self.validator_address_cache.read();
+
+        // This uses unstable method from substrate's API, but there's probably no other easy way
+        // of doing this. On the other hand, the p2p peer_id is only for debuging purposes,
+        // so in case of future substrate API change this if statement can be temporarily safely removed.
+        if let Ok(network_state) = self.network.network_state().await {
+            attach_p2p_network_info_to_validator_addressing_info(&mut info, network_state);
```

Comment:
I think it's OK to drop this, unless you think it gives good results...

Is it the case that using a combination of the new RPC call (without `potential_p2p_network_additional_info`) and the old `system_unstable_networkState` one could recover roughly the same info?

Reply:
Combining these two rpc calls gives exactly the same info. I think I can remove p2p peerid, move the joining part to some python script and put it somewhere in our internal documentation.

Reply:
Yes, so maybe you can indeed drop p2p peerid. 

There is also this issue that sometimes nodes use domain names and not ips, which complicates matters :)

Reply:
Dropped